lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -85,5 +97,51 @@ testRule({
 			endColumn: 47,
 			description: 'an unknown descriptor in a nested at-rule',
 		},
+		{
+			code: '@page { bar: 0; }',
+			message: messages.rejected('@page', 'bar'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 12,
+			description: 'an unknown descriptor or page context property in @page',
+		},
+		{
+			code: '@page { content: "foo"; }',
+			message: messages.rejected('@page', 'content'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 16,
+			description: 'a margin context property in @page',
+		},
+		{
+			code: '@page { @top-center { bar: 0; } }',
+			message: messages.rejected('@page', 'bar'),
+			line: 1,
+			column: 23,
+			endLine: 1,
+			endColumn: 26,
+			description: 'an unknown descriptor or margin context property in a margin at-rule',
+		},
+		{
+			code: '@page { foo: 0; @top-center { bar: 0; } }',
```

Comment:
That's a good test!

Reply:
I agree.
i.e. once `css-tree` will support margin at-rules, it will be triggered